kpops/components/base_components/kafka_connect.py

```diff
@@ -72,6 +72,11 @@ def helm_diff(self) -> HelmDiff:
     def kafka_connect_resetter_chart(self) -> str:
         return f"{self.repo_config.repository_name}/kafka-connect-resetter"
 
+    def set_connector_name(self):
```

Comment:
Only do that in the connect_handler where the validation is called

Reply:
but why not in the origin ? at the end it is something mandatory, right ?

Reply:
It's not mandatory for the endpoints we call except the validation afaik